feed-integration-webapp/src/main/resources/atomfeed.properties

```diff
@@ -0,0 +1,15 @@
+#Server properties
+feed.author=OpenMRS
+
+#Client Properties
+feed.connectionTimeoutInMilliseconds=10000
+feed.replyTimeoutInMilliseconds=20000
+feed.maxFailedEvents=10000
+feed.failedEventMaxRetry=10
+#OpenMRS data specific properties
+openmrs.auth.uri=http://localhost:8080/openmrs/ws/rest/v1/session
+openmrs.patient.feed.uri=http://localhost:8080/openmrs/ws/atomfeed/patient/recent
+openmrs.user=superman
```

Comment:
Crater bridge should not use `superman` to talk to openmrs. Best to use a crater specific credentials string. We don't want crater calls to look like these were "user" made calls to OpenMRS.

Reply:
Waiting on response from Mohan about how a new user can be created automatically when running the crater-atomfeed docker image with bahmni docker image.